pom.xml

```diff
@@ -23,8 +22,8 @@
     <properties>
         <commonsCodecVersion>1.11</commonsCodecVersion>
         <fluentHcVersion>4.5.2</fluentHcVersion>
-        <grizzlyVersion>2.3.36-MULE-006</grizzlyVersion>
-        <grizzlyAhcVersion>1.14-MULE-007</grizzlyAhcVersion>
+        <grizzlyVersion>2.3.36-MULE-007-SNAPSHOT</grizzlyVersion>
```

Comment:
do we really need to go through the snapshot here, or should we just release the dependency and change this dependency to a release directly?